src/components/LocalePicker/index.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
```

Comment:
Commenting here because I can't comment on the file name itself. I think this file should be `src/components/LocalePicker.js`, no? Since we only have one version of it, we don't need the folder, right?

Reply:
You're right. I have moved `components/LocalePicker/index.js` to `components/LocalePicker.js`. Thanks.